src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataUpdater.java

```diff
@@ -58,6 +70,26 @@
         return updatedCaseData;
     }
 
+    @SuppressWarnings("unchecked")
+    public Map<String, Object> updateUpdateEnvelopeReferences(
```

Comment:
updateEnvelopeReferencesForCaseUpdate ?

Reply:
done

Reply:
If we have CaseAction then can we just say updateEnvelopeReferences ?